src/Domain.LinnApps/ImportBooks/ImportBookService.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace Linn.Stores.Domain.LinnApps.ImportBooks
+{
+    using System.Collections.Generic;
+    using Linn.Common.Persistence;
+    using Linn.Stores.Domain.LinnApps.Parts;
+
+    public class ImportBookService : IImportBookService
+    {
+        private readonly IRepository<ImportBookInvoiceDetail, ImportBookInvoiceDetailKey> InvoiceDetailRepository;
+        private readonly IRepository<ImportBookOrderDetail, ImportBookOrderDetailKey> OrderDetailRepository;
```

Comment:
should be invoiceDetailRepository rather than InvoiceDetailRepository although i don't think you actually need them at all